tests/sentry/utils/test_json.py

```diff
@@ -48,3 +49,13 @@ def test_enum(self):
 
     def test_translation(self):
         self.assertEqual(json.dumps(_("word")), '"word"')
+
+    @patch("sentry_sdk.start_span")
+    def test_loads_with_sdk_trace(self, sentry_sdk_mock):
```

Comment:
Nit:

```suggestion
    def test_loads_with_sdk_trace(self, start_span_mock):
```

It took me a second to look above the signature and figure out what was being mocked so that the test made sense.